app/src/main/java/com/nextcloud/talk/contacts/components/SearchComponent.kt

```diff
@@ -101,3 +109,10 @@ fun searchKeyboardActions(text: String, keyboardController: SoftwareKeyboardCont
             }
         }
     )
+
+class SearchComponentCompanionClass {
+    companion object {
+        const val WIDTH_RATI0_SMALL = 0.85f
```

Comment:
This width is aligned to the length of the text string "Add", which won't work for other languages.
Changed the translation to "Addddddddddddd1" and it looks like this:

![grafik](https://github.com/user-attachments/assets/313cf4d9-a814-4c38-8a91-3738198525ae)